cmd/mesh/manifest-apply.go

```diff
@@ -43,8 +43,8 @@ type manifestApplyArgs struct {
 func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config.")
-	cmd.PersistentFlags().DurationVar(&args.readinessTimeout, "readiness-timeout", 300*time.Second, "Maximum time to wait for all Istio resources to be ready."+
-		"--wait must be set for this flag to apply.")
+	cmd.PersistentFlags().DurationVar(&args.readinessTimeout, "readiness-timeout", 300*time.Second, "Maximum time in seconds to wait for all Istio resources to be ready."+
```

Comment:
This line exceeds the maximum line length of 160 characters, see: https://github.com/istio/operator/blob/master/common/config/.golangci.yml#L80